java-checks/src/test/java/org/sonar/java/checks/AbstractClassNoFieldShouldBeInterfaceCheckTest.java

```diff
@@ -23,25 +23,30 @@
 import org.junit.jupiter.api.Test;
 import org.sonar.java.checks.verifier.JavaCheckVerifier;
 
+import static org.assertj.core.api.AssertionsForClassTypes.assertThat;
 import static org.junit.jupiter.api.Assertions.assertFalse;
```

Comment:
`assertTrue` and `assertFalse` are not used anymore and the imports should be dropped.